subsys/bluetooth/host/mesh/settings.c

```diff
@@ -897,22 +867,20 @@ void bt_mesh_store_net(void)
 
 static void store_pending_iv(void)
 {
-	char buf[BT_SETTINGS_SIZE(sizeof(struct iv_val))];
 	struct iv_val iv;
-	char *str;
+	int err;
 
 	iv.iv_index = bt_mesh.iv_index;
 	iv.iv_update = bt_mesh.iv_update;
 	iv.iv_duration = bt_mesh.ivu_duration;
 
-	str = settings_str_from_bytes(&iv, sizeof(iv), buf, sizeof(buf));
-	if (!str) {
-		BT_ERR("Unable to encode IV as value");
-		return;
+	err = settings_save_one("bt/mesh/IV", &iv, sizeof(iv));
+	if (err) {
+		BT_DBG("Failed to store IV value");
+	} else {
+		BT_DBG("Stored IV value");
+		BT_HEXDUMP_DBG(&iv, sizeof(iv), "raw value");
```

Comment:
I don't think it's worth doing the he hexdump for the raw value. Previously the code was logging the base64 value so I could compare it to what was restored, however the raw value provides little, well, value, here :)